common/src/main/java/com/datastrato/gravitino/dto/requests/TableUpdateRequest.java

```diff
@@ -423,6 +427,60 @@ public TableChange tableChange() {
     }
   }
 
+  /** Represents a request to update the default value of a column of a table. */
+  @EqualsAndHashCode
+  @ToString
+  class UpdateTableColumnDefaultValueRequest implements TableUpdateRequest {
+
+    @Getter
+    @JsonProperty("fieldName")
+    private final String[] fieldName;
+
+    @Getter
+    @JsonProperty("newDefaultValue")
+    @JsonSerialize(using = JsonUtils.ColumnDefaultValueSerializer.class)
+    @JsonDeserialize(using = JsonUtils.ColumnDefaultValueDeserializer.class)
+    private final Expression newDefaultValue;
+
+    /**
+     * Constructor for UpdateTableColumnDefaultValueRequest.
+     *
+     * @param fieldName the field name to update
+     * @param newDefaultValue the new default value of the field
+     */
+    public UpdateTableColumnDefaultValueRequest(String[] fieldName, Expression newDefaultValue) {
+      this.fieldName = fieldName;
+      this.newDefaultValue = newDefaultValue;
+    }
+
+    /** Default constructor for Jackson deserialization. */
+    public UpdateTableColumnDefaultValueRequest() {
+      this(null, null);
+    }
+
+    /**
+     * Validates the request.
+     *
+     * @throws IllegalArgumentException If the request is invalid, this exception is thrown.
+     */
+    @Override
+    public void validate() throws IllegalArgumentException {
+      Preconditions.checkArgument(
+          fieldName != null
+              && fieldName.length > 0
+              && Arrays.stream(fieldName).allMatch(StringUtils::isNotBlank),
+          "\"fieldName\" field is required and cannot be empty");
+      Preconditions.checkArgument(
+          newDefaultValue != null, "\"newDefaultValue\" field is required and cannot be empty");
```

Comment:
Is `DEFAULT_VALUE_NOT_SET` allowed?

Reply:
I add [a test for DEFAULT_VALUE_NOT_SET validation](https://github.com/datastrato/gravitino/pull/2673/commits/a43efcd9cf9895c18722fea9852283595622f770)  , it's allowed.

Reply:
If doing so is meaningless, then I suggest throwing an exception during validation.

Reply:
Updated.